.github/workflows/ci.yml

```diff
@@ -19,12 +19,12 @@ permissions:
   contents: read
 
 env:
-  DEVELOPER_DIR: /Applications/Xcode_14.2.app
+  DEVELOPER_DIR: /Applications/Xcode_14.3.app
 
 jobs:
   build:
     name: Build for ${{ matrix.project }}
-    runs-on: macos-12
+    runs-on: macos-13
```

Comment:
GitHub Actions support for macOS Ventura, which is required to run Xcode 14.3, has been delayed due to changes in Apple's hardware compatibility requirements. The environment is currently not available for GitHub Actions.

https://github.com/actions/runner-images/issues/6426#issuecomment-1479974889

Reply:
I know. But I went ahead and Bumped it because I couldn't get the CI through anyway.

Reply:
Xcode 14.3 (macOS Ventura) is now available at GitHub Actions.

- GitHub Actions: macOS 13 is now available https://github.blog/changelog/2023-04-24-github-actions-macos-13-is-now-available